source/tests/universal/pt/model/test_model.py

```
@@ -803,3 +805,100 @@
+        dp_model1 = DPAtomicModel(
+            ds1,
+            ft1,
+            type_map=cls.expected_type_map,
+        )
+        dp_model2 = DPAtomicModel(
+            ds2,
+            ft2,
+            type_map=cls.expected_type_map,
+        )
+        cls.module = DPLinearModel(
+            dp_model,
```

Ruff

Undefined name `dp_model`

(F821)